site/smart-accounts/modular-account/index.md

```diff
@@ -6,84 +6,52 @@ head:
 
-Read on to learn more about ERC-6900 and modular accounts.
+## Why Modular Account?
 
-:::tip Note
-We are working towards the first stable version of [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) with [the community](https://ethereum-magicians.org/t/erc-6900-modular-smart-contract-accounts-and-plugins/13885). If you're developing a plugin or modular account, we'd love to chat!
+### Make the most of Account Abstraction
 
-Please join the waitlist [here](https://docs.google.com/forms/d/1Z3wFRiMoEKoo8FJFrymVEOzrbKQXjSnYhm_hKKDnooE/edit). You can also join the modular accounts [Telegram group](https://t.me/+KfB9WuhKDgk5YzIx) or [email](mailto:account-abstraction@alchemy.com) us!
-:::
+Smart accounts unlocks lots of customizable ways to improve the wallet experience, but it requires writing this behavior into the smart contract for the account, which is difficult and security-critical. Modular Account uses the [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) framework to simplify creating powerful features for smart accounts. We’ve created two plugins to level up your smart accounts, and look forward to what new plugins you create!
 
-## Motivation
+### Multi Owner Plugin
 
-In the coming years, we expect most user accounts to be smart accounts that leverage the benefits of account abstraction. These accounts will generally share a similar set of core features such as signature validation and ownership transfer. To ensure this core feature set is secure and does not contain any vulnerabilities, it will be prudent for most developers to re-use battle-tested smart accounts rather than writing their own accounts.
+The Multi Owner plugin lets your smart accounts have one or more ECDSA or SCA owners. This lets you account integrate with multiple signers at once, and supports recovering your account if one signer is lost.
 
-However, smart acounts are also programmable, enabling you to build new and innovative features that hook into the validation and execution logic of a smart account. We hope and expect to see a diverse ecosystem of plugins flourish.
+Read more about Multi Owner Plugin and how to get started with Modular Account [here]()!
 
-In order to maximize interoperability and code re-use, these plugins will ideally share a standard interface that is compatible with every smart account.
+### Session Key Plugin
 
-Refer to the Account Abstraction [series](https://www.alchemy.com/blog/account-abstraction/?a=ak-docs) on our blog to learn more.
+The Session Key plugin lets your smart account add additional signers to your account with specific permissions. These can be keys that may only interact with a specific dapp, spend up to a set amount of ERC-20 tokens, or expire after 30 minutes. Session keys let you streamline interactions by reducing confirmation steps, or automate actions on behalf of the account. And these features are kept secure through the permission system, which protects the account from malicious use of the session key.
```

Comment:
It might be helpful to be more clear + direct about flexibilities of session keys.

```suggestion
The Session Key plugin lets your smart account add additional signers to your account with specific permissions. 

Session keys can be customized and configured to:
- Dapp Restrictions: restrict to only interact with specific dapp(s)
- Spending Limits: spend up to a set amount of ERC-20 tokens or native token amount
- Time Period: Expire after certain time periods.

Session keys let you streamline interactions by reducing confirmation steps, or automate actions on behalf of the account. And these features are kept secure through the permission system, which protects the account from malicious use of the session key.
```

Reply:
+1 to this recommendation with a couple small tweaks:
I would change `Dapp Restrictions` to actually be:

```
Contract Restrictions: restrict to only interact with specific contracts and/or a subset of their methods
```